src/core/control/XournalMain.h

```diff
@@ -12,8 +12,11 @@
 #pragma once
 
 
+#include <glib.h>
 
-
+#include "gui/GladeSearchpath.h"
 namespace XournalMain {
 auto run(int argc, char** argv) -> int;
-}
+
+void initResourcePath(GladeSearchpath* gladePath, const gchar* relativePathAndFile, bool failIfNotFound = true);
```

Comment:
```suggestion
void initResourcePath(GladeSearchpath* gladePath, std::string_view relativePathAndFile, bool failIfNotFound = true);
```
Is this function required by other compilation units?